### PR TITLE
Closes #4622: Add unit tests for extensions

### LINKED ISSUE #4622: Create unit tests for Extensions
We should write unit tests for the extension functions we use. This ticket will encompass writing tests for all of the extensions. If we feel this is too large for one ticket, let's break a larger component out. 

* Activity
* BookmarkNode
* BrowserIcons
* Context
* Drawable
* Fragment
* Log
* NavController
* Resources
* Session
* String
* TabCollection
* View

### PR DESCRIPTION
**Pull request by NotWoods**

### CODE REVIEW

File: app/src/test/java/org/mozilla/fenix/ext/AccessibilityManagerTest.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package org.mozilla.fenix.ext
```
Comment (ekager) [inline app/src/test/java/org/mozilla/fenix/ext/AccessibilityManagerTest.kt:1]: nit - license

**Review by ekager — CHANGES_REQUESTED** [review]
I think there are some merge conflicts in the BaseBrowserFragment :/

**Comment by eliserichards** [comment]
bors try

**Comment by eliserichards** [comment]
bors try

**Comment by eliserichards** [comment]
bors try

**Comment by jonalmeida** [comment]
It's failing at:

```
[task 2020-10-05T20:43:32.650Z] > Task :app:compileDebugUnitTestKotlin FAILED
[task 2020-10-05T20:43:32.650Z] e: /builds/worker/checkouts/src/app/src/test/java/org/mozilla/fenix/ext/LiveDataTest.kt: (22, 18): Unresolved reference: observeOnceAndRemoveObserver
[task 2020-10-05T20:43:32.749Z] 
[task 2020-10-05T20:43:32.749Z] FAILURE: Build failed with an exception.
[task 2020-10-05T20:43:32.749Z] 
[task 2020-10-05T20:43:32.749Z] * What went wrong:
[task 2020-10-05T20:43:32.749Z] Execution failed for task ':app:compileDebugUnitTestKotlin'.
[task 2020-10-05T20:43:32.749Z] > Compilation error. See log for more details
[task 2020-10-05T20:43:32.749Z] 
```

**Comment by eliserichards** [comment]
The first time it was failing on the UI tests :)

**Comment by eliserichards** [comment]
We're still getting a 403 from test-debug. Re-running bors

**Comment by eliserichards** [comment]
bors try

**Comment by eliserichards** [comment]
bors try

**Comment by liuche** [comment]
@JohanLorenzo it seems like using bors to run try doesn't seem to be working here. Do you have any ideas on why this might be?

**Comment by liuche** [comment]
Oh, looks like a build error?
`e: fenix/app/src/test/java/org/mozilla/fenix/ext/LiveDataTest.kt: (22, 18): Unresolved reference: observeOnceAndRemoveObserver`

**Comment by liuche** [comment]
green green green green green

**Review by eliserichards — APPROVED** [review]
👍 yay, tests!

**Comment by eliserichards** [comment]
We do not want to make changes to the BaseBrowserFragment now, so closing this.

**Pull request closed without merging**